inst/project/example/_targets.R

```diff
@@ -0,0 +1,44 @@
+library(targets)
+library(dplyr)
+library(glue)
+library(rlang)
+source(here::here("inst/project/utils.R"))
+source(here::here("inst/project/targets.R"))
```

Comment:
I think the inclusion of the entirety of this file is a bit out-of-scope. It includes helper functions that aren't used in the pipeline and will probably cause some confusion to their relevance.

Reply:
Yeah, that makes sense, I'll keep just the bare minimum!!